### PR TITLE
short description on next line

### PR DESCRIPTION
**Pull request by Zac-Rimmer**
:checkered_flag:

### CODE REVIEW

**Comment by Zac-Rimmer** [comment]
@rsilcock @regatta-sab @regatta-sd Please could somebody sign off :smile:

File: hawkshead/css/hawkshead-nt.css [hunk 1]
```diff
@@ -3600,6 +3600,8 @@ a:nth-child(4) .offers-box {
 
 }
 
+<<<<<<< HEAD
```
Comment (rsilcock) [inline hawkshead/css/hawkshead-nt.css]: Is this a typo?

Comment (Zac-Rimmer) [inline hawkshead/css/hawkshead-nt.css]: Missed a conflict comment :expressionless:

**Comment by Zac-Rimmer** [comment]
Updated :+1:

**Pull request merged**